File: portal_backend/src/entities/agile_story.ts
import {
  Entity,
  PrimaryGeneratedColumn,
  Column,
  ManyToOne,
  OneToMany,
  JoinColumn
} from 'typeorm'
import { AgileSprint } from '../entities/agile_sprint'
import { AgileStoryAgileSystemUser } from '../entities/agile_story_agile_system_user'
@Entity({ name: 'agile_story' })
export class AgileStory {
  @PrimaryGeneratedColumn('uuid', { name: 'agile_story_id' })
  agileStoryId: string

  @Column('varchar', { name: 'agile_story_name' })
  agileStoryName: string

  @ManyToOne(type => AgileSprint)
  @JoinColumn({ name: 'agile_sprint_id' })
  agileSprint: AgileSprint

  @OneToMany(type => AgileStoryAgileSystemUser, agileStoryAgileSystemUser => agileStoryAgileSystemUser.agileStory)
  agileStoryAgileSystemUsers: AgileStoryAgileSystemUser[]

  @Column('varchar', { name: 'story_description' })
  storyDescription: string

  @Column('varchar', { name: 'story_type' })
  storyType: string

  @Column('varchar', { name: 'story_status' })
  storyStatus: string

  @Column('varchar', { name: 'story_points' })
  storyPoints: string
}

File: fg_metrics/VSTS/VSTSComms/VSTSComms/AgileStory.cs
using System;
using System.Collections.Generic;

namespace VSTSComms
{
    public partial class AgileStory
    {
        public string AgileStoryId { get; set; }
        public string AgileStoryName { get; set; }
        public string AgileSprintId { get; set; }
        public string StoryDescription { get; set; }
        public string StoryType { get; set; }
        public string StoryStatus { get; set; }
        public string StoryPoints { get; set; }
    }
}
